docs/src/intro/publications.md

```diff
@@ -164,8 +168,20 @@ Wang, X., Zhang, J., Babovic, V., 2016. Improving real-time forecasting of water
 indicators with combination of process-based models and data assimilation technique.
 Ecological Indicators 66, 428–439. <https://doi:10.1016/j.ecolind.2016.02.016>.
 
+### Datasets
```

Comment:
Thanks for updating the list of publications @aweerts . Nice to also list datasets. I would suggest to only list these if the dataset is not part of a report or paper (to minimize the work to keep track of these datasets and updating the list) , otherwise I think adding the report/paper is fine. Are/will these datasets be part of a publication?
 
@JoostBuitink , @dalmijn, @alimeshgi: what do you think, do you have a preference?

Reply:
I agree to only list a dataset if it is not part of a publication.

Maybe it is nice to move the datasets to the bottom of this page, to put more emphasis on the reports?

Reply:
yes the datasetswill be part of publications (under review)
Hydrological Reference Dataset (Tier 1) for the ESA 4DHydro Open Science Catalogue
Droppers et al. 2024, submitted to Scientific Data